modules/network-observability-deploy-network-policy.adoc

```diff
@@ -0,0 +1,43 @@
+// Module included in the following assemblies:
+
+// * networking/network_observability/network-observability-network-policy.adoc
+
+
+:_mod-docs-content-type: PROCEDURE
+[id="network-observability-deploy-network-policy_{context}"]
+= Configuring an ingress network policy by using the FlowCollector custom resource
+You can configure the `FlowCollector` custom resource to deploy an ingress Network Policy for Network Observability by setting the `spec.NetworkPolicy.enable` specification in  to `true`. By default, the specification is `false`.
+
+Ensure that Network Observability components you have configured can communicate with components that are external to the namespace, which might include:
+
+   * Connection to Loki (as defined in `FlowCollector` `spec.loki`)
+   * Connection to Kafka (as defined in `FlowCollector` `spec.kafka`)
+   * Connection to any exporter (as defined in FlowCollector `spec.exporters`)
+   * If you are using Loki and including it in the policy target, connection to an external object storage (as defined in your `LokiStack` related secret)
+
+If you need further isolation for egress traffic, you can create an additional policy that. For more information, see 
```

Comment:
missing `for` after policy

Reply:
I actually need to remove this entire thing because we're not going to recommend that anymore, per Joel's comment here: https://issues.redhat.com/browse/OSDOCS-11555